addon/list-utils.js

```diff
@@ -133,7 +133,9 @@ export function getItemsFromAjaxCall ({ajax, bunsenId, data, filter, options, va
  */
 export function getItemsFromEmberData (value, modelDef, data, bunsenId, store, filter) {
   const modelType = modelDef.modelType || 'resources'
-  const {labelAttribute, valueAttribute} = modelDef
+  const {labelAttribute, queryForCurrentValue, valueAttribute} = modelDef
+  const valueAsId = Number(value[bunsenId])
+  const actuallyFindCurrentValue = queryForCurrentValue && !!valueAsId
```

Comment:
I'm not sure I see the need to have `queryForCurrentValue`. Is there a reason not to just always query for the current value? If you can think of a use case in which it'd break then maybe make it opt-out instead of opt-in?